skaha/src/main/java/org/opencadc/skaha/utils/CommonUtils.java

```diff
@@ -24,10 +27,46 @@ public static String encodeBase64(final byte[] encodedValue) {
      * Obtain the first configured Service URI for the given base standard ID.
      *
      * @param baseStandardID The URI to lookup.
-     * @return A single URI (first matching).  Never null.
+     * @return A single URI (first matching). Never null.
      */
     public static URI firstLocalServiceURI(final URI baseStandardID) {
         final Set<URI> serviceURIs = new LocalAuthority().getServiceURIs(baseStandardID);
         return serviceURIs.stream().findFirst().orElseThrow(IllegalStateException::new);
     }
+
+    /**
+     * Format to Binary SI values.
+     *
+     * @param bytes Bytes reported from Kubernetes.
+     * @return String formatted, never null.
+     */
+    public static String formatMemoryFromBytes(final BigDecimal bytes) {
```

Comment:
Could we use `FileUtils.byteCountToDisplaySize(long size)` from `org.apache.commons.io`. 

https://commons.apache.org/proper/commons-io/javadocs/api-2.4/org/apache/commons/io/FileUtils.html#byteCountToDisplaySize(long)

Reply:
Yes, but this actually isn't used and I'm removing it.  Once my `Kubernetes-java` client changes are accepted, I'll use that.